layer.yaml

```diff
@@ -21,7 +21,7 @@ defines:
       to help avoid conflicts with other charms or libraries on the machine.
   include_system_packages:
     type: boolean
-    default: true
+    default: false
```

Comment:
I agree that `false` is a safer default, but the reason we chose `true` in the first place was to mitigate the impact of changing the default of `use_venv` to `true`.  Probably it would be sufficient to simply be vocal about this change to let people know that it could impact them.  I suspect the number of charms that would have an issue with this would be pretty small.

Reply:
https://discourse.jujucharms.com/t/heads-up-change-of-default-in-layer-basic/1562